Sources/SnapshotTesting/Strategy/Any.swift

```diff
@@ -35,6 +38,7 @@ private func snap<T>(_ value: T, name: String? = nil, indent: Int = 0) -> String
   case (_, .class?), (_, .struct?):
     description = String(describing: mirror.subjectType)
       .replacingOccurrences(of: " #\\d+", with: "", options: .regularExpression)
+    children = sort(children)
```

Comment:
I was doubtful at first to introduce this, as it will force people regenerate a lot of snapshots, but I think that, even if member-wise inits change when the order of the fields changes, the snapshot content shouldn't be aware of this, as the content is really the same. What do you think?

Reply:
Ha, at first thought I was doubtful but after thinking it through I'm coming around to it. Let's do it for now and we can always revisit if there are any consequences.